### PR TITLE
[MIG] sale_invoice_group_method: Migration to 12.0

### PR DESCRIPTION
**Pull request by rven**

### CODE REVIEW

**Comment by rven** [comment]
Depends on https://github.com/OCA/sale-workflow/pull/774

File: sale_invoice_group_method/tests/test_sale_invoice_group_method.py [hunk 1]
```diff
@@ -0,0 +1,297 @@
...
+        so2.onchange_partner_id()
+        self.assertEquals(so2.invoice_group_method_id,
+                          self.sale_invoice_group_method,
+                          "The invoice group method should be"
+                          "'sale_invoice_group_method'")
+        so2 = self._create_sale_order_lines(so2)
+        inv1 = self._create_invoice_from_sale(so1, False)
+        inv2 = self._create_invoice_from_sale(so2, False)
+        # The result is two different invoices (one for each sale order)
+        self.assertTrue(inv1 != inv2)
```
Comment (nikul-serpentcs) [inline sale_invoice_group_method/tests/test_sale_invoice_group_method.py]: @rven Use` assertNotEqual`

Comment (rven) [reply sale_invoice_group_method/tests/test_sale_invoice_group_method.py]: Done!

**Comment by ghost** [comment]
Hello,
Is someone in charge of this at this time or has it been abandoned?
Thanks!

**Comment by rven** [comment]
Rebase done!

**Review by rousseldenis — APPROVED** [review]
Code review

**Comment by ghost** [comment]
I have created a invoice group method and when I have tried to assign it to a customer this traceback has appeared

Error:
Odoo Server Error

Traceback (most recent call last):
  File "/.repo_requirements/odoo/odoo/http.py", line 654, in _handle_exception
    return super(JsonRequest, self)._handle_exception(exception)
  File "/.repo_requirements/odoo/odoo/http.py", line 312, in _handle_exception
    raise pycompat.reraise(type(exception), exception, sys.exc_info()[2])
  File "/.repo_requirements/odoo/odoo/tools/pycompat.py", line 87, in reraise
    raise value
  File "/.repo_requirements/odoo/odoo/http.py", line 696, in dispatch
    result = self._call_function(**self.params)
  File "/.repo_requirements/odoo/odoo/http.py", line 344, in _call_function
    return checked_call(self.db, *args, **kwargs)
  File "/.repo_requirements/odoo/odoo/service/model.py", line 97, in wrapper
    return f(dbname, *args, **kwargs)
  File "/.repo_requirements/odoo/odoo/http.py", line 337, in checked_call
    result = self.endpoint(*a, **kw)
  File "/.repo_requirements/odoo/odoo/http.py", line 939, in __call__
    return self.method(*args, **kw)
  File "/.repo_requirements/odoo/odoo/http.py", line 517, in response_wrap
    response = f(*args, **kw)
  File "/home/odoo/OCB-12.0/addons/web/controllers/main.py", line 962, in call_kw
    return self._call_kw(model, method, args, kwargs)
  File "/home/odoo/OCB-12.0/addons/web/controllers/main.py", line 954, in _call_kw
    return call_kw(request.env[model], method, args, kwargs)
  File "/.repo_requirements/odoo/odoo/api.py", line 749, in call_kw
    return _call_kw_multi(method, model, args, kwargs)
  File "/.repo_requirements/odoo/odoo/api.py", line 736, in _call_kw_multi
    result = method(recs, *args, **kwargs)
  File "/home/odoo/OCB-12.0/addons/partner_autocomplete/models/res_partner.py", line 183, in write
    res = super(ResPartner, self).write(values)
  File "/.repo_requirements/odoo/odoo/addons/base/models/res_partner.py", line 575, in write
    partner._fields_sync(vals)
  File "/.repo_requirements/odoo/odoo/addons/base/models/res_partner.py", line 502, in _fields_sync
    self._children_sync(values)
  File "/.repo_requirements/odoo/odoo/addons/base/models/res_partner.py", line 510, in _children_sync
    if any(field in values for field in commercial_fields):
TypeError: 'NoneType' object is not iterable

**Comment by ghost** [comment]
Hello,
Is there news about it?
Thanks.

**Comment by robin5210** [comment]
Hi,

With the new rebase all tests are passing and code looks fine. 
Could you test again @gelo-landoo?

Kind regards,
Robin

**Comment by ghost** [comment]
Ok,
I'll test it ASAP and I tell you the results.
Thanks.

**Comment by ghost** [comment]
Hello @robin5210 ,
I have tested it and it works properly, but I don't know how to mark the test as passed

**Comment by LoisRForgeFlow** [comment]
Hi @gelo-landoo 

Go to "Files changed" tab of this PR and in the top right corner of the tab you will see a green button "Review Changes", you can approve them there.

**Review by ghost — APPROVED** [review]
I have checked and it runs proprerly

**Comment by ghost** [comment]
Thanks @lreficent

**Comment by robin5210** [comment]
Is there anything that needs to be done before this can be merged?

**Comment by rven** [comment]
@pedrobaeza Could this be merged?

**Pull request merged**